projects/serializers.py

```diff
@@ -86,7 +86,7 @@ class ProjectControlSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = ProjectControl
-        fields = ("status", "project", "control", "catalog_data", "component_data", )
+        fields = ("status", "project", "control", "remarks", "catalog_data", "component_data", )
```

Comment:
Should probably add validation somewhere so that if someone tries to set "status" to NA, but doesn't provide "remarks", that they'll get a 400. I think that was a requirement right (that you _had_ to provide a reason if NA)?

Reply:
That is a requirement, but I hadn't thought of enforcing it on the back end. Good call!